documentation/usage.md

```diff
@@ -167,6 +167,52 @@ old_sum_behavior = configs.sum_default_zero
 configs.sum_default_zero = False
 ```
 
+The following configs are used in the behavior of `PERCENTILE`, `RANKING`, `ORDER_BY` and `TOP_K` where a collation(`ASC` or `DESC`) is not explicitly specified:
+
+3. `collation_default_asc` (default=True): if True, then the default collation is ascending. If False, then the default collation is descending.
+4. `propogate_collation` (default=False): if True, then the collation of the current expression, which does not have a collation, uses the most recent available collation in the nodes of the term. If there is no recent available collation, then the default collation is used as specified by `collation_default_asc`. If False, the expression uses the default collation as specified by `collation_default_asc`.
+
+For example, consider the following PyDough code:
+
+```py
+%%pydough
+# The collations are not explicitly specified for few of the terms.
+Suppliers.ORDER_BY(
+  COUNT(lines),
+  nation.name.ASC(),
+  COUNT(supply_records),
+  account_balance.DESC(),
+  key,
+)
+
+# Let's see the behavior of the terms with different configurations
+# With the default settings
+Suppliers.ORDER_BY(
+  COUNT(lines).ASC(),
+  nation.name.ASC(),
+  COUNT(supply_records).ASC(),
+  account_balance.DESC(),
+  key.ASC(),
+)
+
+# With collation_default_asc=False and propogate_collation=False
+Suppliers.ORDER_BY(
+  COUNT(lines).DESC(),
+  nation.name.ASC(),
+  COUNT(supply_records).DESC(),
+  account_balance.DESC(),
+  key.DESC(),
+)
+
+# With collation_default_asc=True and propogate_collation=True
+Suppliers.ORDER_BY(
+  COUNT(lines).ASC(),
+  nation.name.ASC(),
+  COUNT(supply_records).ASC(),
+  account_balance.DESC(),
+  key.DESC(),
+)
+```
```

Comment:
nit, add fourth case as well
`collation_default_asc=False and propogate_collation=True`